master/src/main/resources/application.yaml

```
@@ -32,14 +32,22 @@ management:
     web:
       base-path: /api/actuator
       exposure:
-        include: [ "health" ]
+        include: [ "health", "prometheus"]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)